### PR TITLE
Spanish translation

### PR DESCRIPTION
**Pull request by vdo**

### CODE REVIEW

**Comment by NanoAkron** [comment]
Please squash commits

**Comment by vdo** [comment]
@NanoAkron ACK, removed

**Comment by fluffypony** [comment]
Please rebase

**Comment by vdo** [comment]
@fluffypony rebased

**Comment by vdo** [comment]
Reviews by spanish speakers welcome!

**Comment by zhizhongzhiwai** [comment]
i think it's better put the commit to reddit to get reviewed.

File: translations/monero-core_es.ts [hunk 1]
```diff
@@ -221,1153 +221,1161 @@
...
         <location filename="../pages/Settings.qml" line="388"/>
         <source>GUI version: </source>
-        <translation type="unfinished"></translation>
+        <translation>Versión de GUI: </translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="395"/>
         <source>Embedded Monero version: </source>
-        <translation type="unfinished"></translation>
+        <translation>Versión de Monero embedido: </translation>
```
Comment (isocolsky) [inline translations/monero-core_es.ts]: Versión de Monero embebido:

File: translations/monero-core_es.ts [hunk 2]
```diff
@@ -221,1153 +221,1161 @@
...
         <location filename="../pages/Sign.qml" line="479"/>
         <source>Signature</source>
-        <translation type="unfinished"></translation>
+        <translation>Firma</translation>
     </message>
     <message>
         <location filename="../pages/Sign.qml" line="309"/>
         <source>Verify a message or file signature from an address:</source>
-        <translation type="unfinished"></translation>
+        <translation>Verificar un mensaje o fichero de una dirección:</translation>
```
Comment (isocolsky) [inline translations/monero-core_es.ts]: "Verificar un mensaje o fichero **desde** una dirección:"

File: translations/monero-core_es.ts [hunk 3]
```diff
@@ -221,1153 +221,1161 @@
...
         <location filename="../components/TableHeader.qml" line="67"/>
         <source>Block height</source>
-        <translation type="unfinished"></translation>
+        <translation>Altura de bloque</translation>
     </message>
     <message>
         <location filename="../components/TableHeader.qml" line="68"/>
         <source>Amount</source>
-        <translation type="unfinished"></translation>
+        <translation>Cantidad</translation>
```
Comment (isocolsky) [inline translations/monero-core_es.ts]: Importe/Monto

File: translations/monero-core_es.ts [hunk 4]
```diff
@@ -221,1153 +221,1161 @@
...
         <location filename="../pages/Transfer.qml" line="66"/>
         <source>OpenAlias error</source>
-        <translation type="unfinished"></translation>
+        <translation>Error de OpenAlias</translation>
     </message>
     <message>
         <location filename="../pages/Transfer.qml" line="125"/>
         <source>Amount</source>
-        <translation type="unfinished"></translation>
+        <translation>Cantidad</translation>
```
Comment (isocolsky) [inline translations/monero-core_es.ts]: Importe/Monto

File: translations/monero-core_es.ts [hunk 5]
```diff
@@ -221,1153 +221,1161 @@
...
+        <translation>
+ID de pago: </translation>
     </message>
     <message>
         <location filename="../pages/Transfer.qml" line="623"/>
         <source>
 Amount: </source>
-        <translation type="unfinished"></translation>
+        <translation>
+Cantidad: </translation>
```
Comment (isocolsky) [inline translations/monero-core_es.ts]: Importe/Monto

File: translations/monero-core_es.ts [hunk 6]
```diff
@@ -221,1153 +221,1161 @@
...
         <location filename="../pages/Transfer.qml" line="639"/>
         <source>Confirmation</source>
-        <translation type="unfinished"></translation>
+        <translation>Confirmación</translation>
     </message>
     <message>
         <location filename="../pages/Transfer.qml" line="667"/>
         <source>Can&apos;t submit transaction: </source>
-        <translation type="unfinished"></translation>
+        <translation>No se puede presentar la transacción:</translation>
```
Comment (isocolsky) [inline translations/monero-core_es.ts]: "No se puede **enviar** la transacción:"

File: translations/monero-core_es.ts [hunk 7]
```diff
@@ -221,1153 +221,1161 @@
...
-        <translation type="unfinished"></translation>
+        <translation>La cartera no está conectada al daemon.</translation>
     </message>
     <message>
         <location filename="../pages/Transfer.qml" line="748"/>
         <source>Connected daemon is not compatible with GUI. 
 Please upgrade or connect to another daemon</source>
-        <translation type="unfinished"></translation>
+        <translation>El daemon conectado no es compatible con la GUI. 
+Por favor actualize o conecte a otro daemon</translation>
```
Comment (isocolsky) [inline translations/monero-core_es.ts]: Por favor **actualice** o **conéctese** a otro daemon

File: translations/monero-core_es.ts [hunk 8]
```diff
@@ -1377,210 +1385,210 @@ Please upgrade or connect to another daemon</source>
...
         <location filename="../pages/Transfer.qml" line="281"/>
         <source>Resolve</source>
-        <translation type="unfinished"></translation>
+        <translation>Resolver</translation>
     </message>
     <message>
         <location filename="../pages/Transfer.qml" line="342"/>
         <source>16 or 64 hexadecimal characters</source>
-        <translation type="unfinished"></translation>
+        <translation>16 o 64 caracteres hexagesimales</translation>
```
Comment (isocolsky) [inline translations/monero-core_es.ts]: hexadecimales

File: translations/monero-core_es.ts [hunk 9]
```diff
@@ -1377,210 +1385,210 @@ Please upgrade or connect to another daemon</source>
...
         <location filename="../pages/TxKey.qml" line="90"/>
         <source>Verify that a third party made a payment by supplying:</source>
-        <translation type="unfinished"></translation>
+        <translation>Verifique que una tercera parte ha hecho el pago proporcionando:</translation>
     </message>
     <message>
         <location filename="../pages/TxKey.qml" line="94"/>
         <source> - the recipient address</source>
-        <translation type="unfinished"></translation>
+        <translation>- la dirección del recipiente</translation>
```
Comment (isocolsky) [inline translations/monero-core_es.ts]: "- la dirección del **receptor**/beneficiario".

"beneficiario" is used elsewhere in place of "recipient"

File: translations/monero-core_es.ts [hunk 10]
```diff
@@ -1377,210 +1385,210 @@ Please upgrade or connect to another daemon</source>
...
         <location filename="../wizard/WizardConfigure.qml" line="79"/>
         <source>We’re almost there - let’s just configure some Monero preferences</source>
-        <translation type="unfinished"></translation>
+        <translation>Ya casi está - vamos a configurar algunos aspectos de Monero</translation>
     </message>
     <message>
         <location filename="../wizard/WizardConfigure.qml" line="97"/>
         <source>Kickstart the Monero blockchain?</source>
-        <translation type="unfinished"></translation>
+        <translation>Iniciar la cadena de bloques de Monero?</translation>
```
Comment (isocolsky) [inline translations/monero-core_es.ts]: Are we not using "¿" ?

File: translations/monero-core_es.ts [hunk 11]
```diff
@@ -1377,210 +1385,210 @@ Please upgrade or connect to another daemon</source>
...
         <location filename="../wizard/WizardConfigure.qml" line="157"/>
         <source>Allow background mining?</source>
-        <translation type="unfinished"></translation>
+        <translation>Permitir minado en segundo plano?</translation>
     </message>
     <message>
         <location filename="../wizard/WizardConfigure.qml" line="175"/>
         <source>Mining secures the Monero network, and also pays a small reward for the work done. This option will let Monero mine when your computer is on mains power and is idle. It will stop mining when you continue working.</source>
-        <translation type="unfinished"></translation>
+        <translation>Minar securiza la red Monero y además paga una pequeña recompensa por el trabajo hecho. Esta opción permite que Monero mine mientras su computadora está ociosa. El minado se detendrá cuando continúe trabajando con ella.</translation>
```
Comment (isocolsky) [inline translations/monero-core_es.ts]: Not sure "securiza" is a word. I think a better translation is "Minar mejora la seguridad de la red Monero..."

File: translations/monero-core_es.ts [hunk 12]
```diff
@@ -1377,210 +1385,210 @@ Please upgrade or connect to another daemon</source>
...
         <location filename="../wizard/WizardDonation.qml" line="176"/>
         <source>Allow background mining?</source>
-        <translation type="unfinished"></translation>
+        <translation>Permitir minado en segundo plano?</translation>
     </message>
     <message>
         <location filename="../wizard/WizardDonation.qml" line="194"/>
         <source>Mining secures the Monero network, and also pays a small reward for the work done. This option will let Monero mine when your computer is on mains power and is idle. It will stop mining when you continue working.</source>
-        <translation type="unfinished"></translation>
+        <translation>Minar securiza la red Monero y además paga una pequeña recompensa por el trabajo hecho. Esta opción permite que Monero mine mientras su computadora está ociosa. El minado se detendrá cuando continúe trabajando con ella.</translation>
```
Comment (isocolsky) [inline translations/monero-core_es.ts]: See above re "securiza"

File: translations/monero-core_es.ts [hunk 13]
```diff
@@ -1590,207 +1598,208 @@ Please upgrade or connect to another daemon</source>
...
         <location filename="../wizard/WizardFinish.qml" line="82"/>
         <source>Don&apos;t forget to write down your seed. You can view your seed and change your settings on settings page.</source>
-        <translation type="unfinished"></translation>
+        <translation>No olvide escribir su semilla. Puede ver su semillar y cambiar sus opciones en la página de opciones.</translation>
     </message>
     <message>
         <location filename="../wizard/WizardFinish.qml" line="128"/>
         <source>You’re all set up!</source>
-        <translation type="unfinished"></translation>
+        <translation>Ya está preparada!</translation>
```
Comment (isocolsky) [inline translations/monero-core_es.ts]: "Ya está listo!"

File: translations/monero-core_es.ts [hunk 14]
```diff
@@ -1590,207 +1598,208 @@ Please upgrade or connect to another daemon</source>
...
         <source>Success</source>
-        <translation type="unfinished"></translation>
+        <translation>Éxito</translation>
     </message>
     <message>
         <location filename="../wizard/WizardMain.qml" line="378"/>
         <source>The view only wallet has been created. You can open it by closing this current wallet, clicking the &quot;Open wallet from file&quot; option, and selecting the view wallet in: 
 %1</source>
-        <translation type="unfinished"></translation>
+        <translation>La cartera de sólo lectura ha sido creada. Puede abrirla cerrando la cartera actual, haciendo clicl en &quot;Abrir cartera desde fichero&quot; y seleccionando la cartera de sólo lectura en:
```
Comment (isocolsky) [inline translations/monero-core_es.ts]: Small typo: clicl -> click

File: translations/monero-core_es.ts [hunk 15]
```diff
@@ -1854,226 +1864,235 @@ Please upgrade or connect to another daemon</source>
...
         <location filename="../main.qml" line="727"/>
         <source>Error</source>
-        <translation type="unfinished"></translation>
+        <translation>Error</translation>
     </message>
     <message>
         <location filename="../main.qml" line="318"/>
         <source>Couldn&apos;t open wallet: </source>
-        <translation type="unfinished"></translation>
+        <translation>No su pudo abrir el wallet:</translation>
```
Comment (isocolsky) [inline translations/monero-core_es.ts]: "No se pudo abrir la cartera:"

File: translations/monero-core_es.ts [hunk 16]
```diff
@@ -1854,226 +1864,235 @@ Please upgrade or connect to another daemon</source>
...
     <message>
         <location filename="../main.qml" line="511"/>
         <location filename="../main.qml" line="622"/>
         <source>
 
 Amount: </source>
-        <translation type="unfinished"></translation>
+        <translation>
+
+Cantidad:</translation>
```
Comment (isocolsky) [inline translations/monero-core_es.ts]: Importe/Monto

File: translations/monero-core_es.ts [hunk 17]
```diff
@@ -1854,226 +1864,235 @@ Please upgrade or connect to another daemon</source>
...
         <location filename="../main.qml" line="709"/>
         <source>Payment check</source>
-        <translation type="unfinished"></translation>
+        <translation>Comprobación de pago</translation>
     </message>
     <message>
         <location filename="../main.qml" line="714"/>
         <source>This address received %1 monero, but the transaction is not yet mined</source>
-        <translation type="unfinished"></translation>
+        <translation>Esta dirección recibió %1 monero, pero la transiacción no ha sido minada</translation>
```
Comment (isocolsky) [inline translations/monero-core_es.ts]: Small typo: transiacción -> transacción

File: translations/monero-core_es.ts [hunk 18]
```diff
@@ -221,1153 +221,1161 @@
...
         <location filename="../pages/Mining.qml" line="164"/>
         <source>Couldn&apos;t start mining.&lt;br&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>No se pudo empezar el minado.&lt;br&gt;</translation>
     </message>
     <message>
         <location filename="../pages/Mining.qml" line="166"/>
         <source>Mining is only available on local daemons. Run a local daemon to be able to mine.&lt;br&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>El minado sólo es disponible en nodos locales. Arranque un daemon local para poder minar.&lt;br&gt;</translation>
```
Comment (ordtrogen) [inline translations/monero-core_es.ts]: Spanish wikipedia has "demonio" for "daemon"

"... disponible en nodos locales." --> "... disponible en demonios locales."  
"Arranque un demonio ..."

File: translations/monero-core_es.ts [hunk 19]
```diff
@@ -221,1153 +221,1161 @@
...
         <location filename="../pages/Settings.qml" line="119"/>
         <source>Show seed</source>
-        <translation type="unfinished"></translation>
+        <translation>Mostrar semilla</translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="132"/>
         <source>Manage daemon</source>
-        <translation type="unfinished"></translation>
+        <translation>Manejar daemon</translation>
```
Comment (ordtrogen) [inline translations/monero-core_es.ts]: Demonio ? (según wikipedia en español)

File: translations/monero-core_es.ts (hunk 10 above)
Comment (ordtrogen) [inline translations/monero-core_es.ts]: Isn't there a better translation of "kickstart"?

**Comment by ordtrogen** [comment]
Great! Thanks for your effort.

I used google to try to get a feel for what word for "wallet" is used in spanish in cryptocurrency. There seems to be no strong consenus as these were the number of results returned:

"monedero bitcoin"  27 000
"cartera bitcoin"	10 200
"billetera bitcoin"  9 000

I would definitely go with "monedero", with the added bonus it sounds nice with "monero". So let's replace "cartera" with "monedero" in the translation.

File: translations/monero-core_es.ts (hunk 19 above)
Comment (elboletaire) [inline translations/monero-core_es.ts]: +1 for `demonio`

File: translations/monero-core_es.ts (hunk 6 above)
Comment (vdo) [inline translations/monero-core_es.ts]: I needed to differentiate between send and commit, but I agree that **enviar** sound better here.

File: translations/monero-core_es.ts (hunk 10 above)
Comment (vdo) [inline translations/monero-core_es.ts]: Inicializar I would say

File: translations/monero-core_es.ts (hunk 11 above)
Comment (vdo) [inline translations/monero-core_es.ts]: It's **segurizar**

File: translations/monero-core_es.ts (hunk 6 above)
Comment (isocolsky) [inline translations/monero-core_es.ts]: Yes, this is a tough one.

**Comment by vdo** [comment]
Thank you all for your effort!

I agree with most of the changes suggested and I will fix them ASAP.

However, in the case of **cantidad** vs **monto**, it is far more common to use **importe** here in Spain, so I'm going to use this one (I assume @isocolsky is from South America).

The sound of mone(de)ro fits perfectly here @ordtrogen :)

@elboletaire  About **demonio**, I avoided the term because it has too 'evil' connotations and it is never used in IT world in spanish, so I would keep the english word. 
Similar with what happens in german:
https://github.com/monero-project/monero-core/pull/638#discussion_r108797215

File: translations/monero-core_es.ts [hunk 20]
```diff
@@ -1854,226 +1864,235 @@ Please upgrade or connect to another daemon</source>
...
         <location filename="../main.qml" line="992"/>
         <source>Monero</source>
-        <translation type="unfinished"></translation>
+        <translation>Monero</translation>
     </message>
     <message>
         <location filename="../main.qml" line="1033"/>
         <source>send to the same destination</source>
-        <translation type="unfinished"></translation>
+        <translation>enviar a la misma destinación</translation>
```
Comment (zzaappedd) [inline translations/monero-core_es.ts]: enviar al mismo destinatario

File: translations/monero-core_es.ts [hunk 21]
```diff
@@ -1854,226 +1864,235 @@ Please upgrade or connect to another daemon</source>
...
         <location filename="../main.qml" line="432"/>
         <source>Daemon failed to start</source>
-        <translation type="unfinished"></translation>
+        <translation>El daemon ha fallado al iniciar</translation>
     </message>
     <message>
         <location filename="../main.qml" line="433"/>
         <source>Please check your wallet and daemon log for errors. You can also try to start %1 manually.</source>
-        <translation type="unfinished"></translation>
+        <translation>Por favor compruebe errores en su cartera y daemon. También puede probar iniciar %1 manualmente.</translation>
```
Comment (zzaappedd) [inline translations/monero-core_es.ts]: Tal vez: Por favor busque errores en los logs de cartera y daemon.

File: translations/monero-core_es.ts [hunk 22]
```diff
@@ -1854,226 +1864,235 @@ Please upgrade or connect to another daemon</source>
...
-        <translation type="unfinished"></translation>
+        <translation>
+
+Cantidad:</translation>
     </message>
     <message>
         <location filename="../main.qml" line="512"/>
         <location filename="../main.qml" line="623"/>
         <source>
 Fee: </source>
```
Comment (zzaappedd) [inline translations/monero-core_es.ts]: Comisión: </source>

File: translations/monero-core_es.ts [hunk 23]
```diff
@@ -1854,226 +1864,235 @@ Please upgrade or connect to another daemon</source>
...
-        <translation type="unfinished"></translation>
+        <translation>
```
Comment (zzaappedd) [inline translations/monero-core_es.ts]: These 2 lines should be replaced by:  `<translation type="unfinished"></translation>` and the previous one by 
Comisión: `</source>`

File: translations/monero-core_es.ts (hunk 11 above)
Comment (JeisonPacateque) [inline translations/monero-core_es.ts]: "Minar asegura la red Monero..."

**Comment by vdo** [comment]
Changes applied.

**Comment by fluffypony** [comment]
I wouldn't worry about evil connotations too much, since this is pull request 666 :-P

**Pull request merged**